content/java/concepts/strings/terms/intern/intern.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: '.intern()'
+Description: 'Creates an exact copy of a string located in the heap memory and stores it in the string constant pool.'
+Subjects:
+- 'Computer Science'
+Tags: 
+- 'Java'
+- 'Methods'
+CatalogContent: 
+- 'learn-java'
+- 'paths/computer-science'
+---
+
+The **.intern() method** creates an exact copy of a string located in the heap memory and stores it in the string constant pool. With this method, it is possible to optimize memory usage in a Java program by reusing identical string objects.
+
+However, if in the string constant pool exists another string with the same value, a new object won’t be created and the new reference will point to the other string.
+ 
+## **Syntax**
+
+There are two ways to invoke the intern() method. The first one is used with the new keyword. The second one is directly used on a string literal.
+
+```pseudo
```

Comment:
You still need to remove the extra line breaks in the code block. Remove the line breaks at the start and end, remove the line breaks between the comment and the code, and only have one line break between the two examples.